zkevm-circuits/src/evm_circuit.rs

```diff
@@ -390,12 +403,17 @@ impl<F: Field> Circuit<F> for EvmCircuit<F> {
                     exp_table,
                     u8_table,
                     u16_table,
+                    feature_config: params,
                 },
             ),
             challenges,
         )
     }
 
+    fn configure(meta: &mut ConstraintSystem<F>) -> Self::Config {
+        Self::configure_with_params(meta, Default::default())
```

Comment:
halo2 will not use this method when `configure_with_params` is implemented.  I would suggest adding an `unreachable!()` here to make sure this method is not called by accident (and thus guarantee that we're always using the features passed by the user).

Reply:
Fixed here. [8787b3c](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1728/commits/8787b3c321647294fc03bf58d61c1b8bcb613dbd)